src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -111,5 +111,15 @@
             <argument type="service" id="translator"/>
             <argument type="string">%validator.translation_domain%</argument>
         </service>
+
+        <!-- SecurityExtension -->
+        <service id="form.extension.security" class="Symfony\Component\Form\Extension\Security\SecurityExtension">
+            <argument type="service" id="security.user_password_encoder" on-invalid="ignore"/>
+            <tag name="form.type_extension" />
```

Comment:
The target class isn't a form type extension, it's a "form extension" which doesn't need to be registered as service.